README.md

```diff
@@ -22,6 +22,31 @@ In this specific configuration, the API contains:
 - Alternatively, install the Configuration from the [Upbound Marketplace](https://marketplace.upbound.io/configurations/upbound/configuration-aws-eks)
 - Check [examples](/examples/) for example XR(Composite Resource)
 
+## ArgoCD Integration
+
+This configuration supports seamless ArgoCD integration through Crossplane's `publishConnectionDetailsTo` field with templating support, implementing the functionality described in the [official Crossplane design document for external secret stores](https://github.com/crossplane/crossplane/blob/main/design/design-doc-external-secret-stores.md#templating-support-for-custom-secret-values).
```

Comment:
This is design doc and secret templating support is documented under Future consideration https://github.com/crossplane/crossplane/blob/main/design/design-doc-external-secret-stores.md#future-considerations

Effectively, it was never implemented in core Crossplane, please check https://github.com/crossplane/crossplane/blob/release-1.20/internal/xcrd/schemas.go#L148-L185